src/main/scala/org/broadinstitute/dsde/workbench/sam/config/GoogleServicesConfig.scala

```diff
@@ -31,6 +31,11 @@ final case class GoogleServicesConfig(
     groupSyncTopic: String,
     groupSyncSubscription: String,
     groupSyncWorkerCount: Int,
+    disableUsersPollInterval: FiniteDuration,
+    disableUsersPollJitter: FiniteDuration,
+    disableUsersTopic: String,
+    disableUsersSubscription: String,
+    disableUsersWorkerCount: Int,
```

Comment:
I also wouldn't mind seeing this  and the config for `groupSync` broken into separate `PubSubConfig` classes like we have with the `GoogleKeyCacheConfig`, especially if we'd need these same values for any future pubsub topics we might add to Sam

Reply:
Yeah, will do some abstraction into a separate config.